src/main/java/com/fp3/haras/model/ComprovanteEstadia.java

```diff
@@ -0,0 +1,52 @@
+package com.fp3.haras.model;
+
+import java.security.Timestamp;
+
+public class ComprovanteEstadia {
+
+    private int id;
```

Comment:
Change all the Id's to Long instead of int. It will make sense when working with JPA